src/scss/_custom.scss

```diff
@@ -13,7 +13,7 @@
       display: inline-block;
       font-size: 30pt;
       font-family: 'Lobster Two', cursive;
-      color: #064583;
+      color: #064583;     //DARK BLUE
```

Comment:
Rather than just commenting the name for these colors, why not use [named Sass variables](http://sass-lang.com/documentation/file.SASS_REFERENCE.html#variables_) for each?